analyzer/src/main/kotlin/managers/Bower.kt

```diff
@@ -252,6 +253,7 @@ class Bower(
                 id = projectPackage.id,
                 definitionFilePath = VersionControlSystem.getPathInfo(definitionFile).path,
                 authors = projectPackage.authors,
+                copyrightHolders = sortedSetOf(),
```

Comment:
An empty set is the default value. Why have you decided to explicitly set it? I've seen that for other properties, like `concludedLicense`, default values are not explicitly provided.

Reply:
I am aware of that the current default value is to set an empty set. But I am a fan of expressiveness  and I have run a few to oten into the situation that someof changed such a default value in good faith because he "just wanted to solve his problem".

I haven't figured out if there is a hard-an-fast rule in the ORT code base because I observed other places where such a default value was explicitly set.

Bottom line: Is there a real technical or business need to fall back to having this value set though the default?